client.js

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import _ from "lodash";
```

Comment:
We already have the `underscore` dependency, lets not add lodash. It is added as a global at the moment, not ideal, and lodash would generally be better, but it's here for historical reasons and I'm afraid we might break people's apps if we change this now. All you need to do here is just remove this lodash import and remove the dependency.